src/main/java/com/dsec/backend/hateoas/RepoAssembler.java

```diff
@@ -0,0 +1,50 @@
+package com.dsec.backend.hateoas;
+
+import com.dsec.backend.controller.RepoController;
+import com.dsec.backend.entity.Repo;
+import com.dsec.backend.model.repo.RepoUpdateDTO;
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.mediatype.Affordances;
+import org.springframework.hateoas.server.mvc.RepresentationModelAssemblerSupport;
+import org.springframework.http.HttpMethod;
+import org.springframework.stereotype.Component;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@Component
+public class RepoAssembler extends RepresentationModelAssemblerSupport<Repo, Repo> {
```

Comment:
This is alright but don't create assemblers in future tasks because the frontend team does not use them. At first they wanted us to return the links, but now they don't use them.